go/vt/dbconfigs/dbconfigs.go

```diff
@@ -135,69 +135,102 @@ func registerPerUserFlags(dbc *userConfig, userKey string) {
 
 }
 
+// ConnParams contains mysql.connparams for now
```

Comment:
We can add additional methods to this type to faciliate some use cases. I see places where we only need to access fields like `Host` or `DbName`. In such cases, it's not necessary to resolve the credentials. Those will be simplified if we exported them here.